src/modules/market-price/adapters/coingecko/index.ts

```diff
@@ -23,6 +23,7 @@ export const symbolIdMap = {
   arb: "arbitrum",
   usdb: "usdb",
   lsk: "lisk",
+  usdzc: "usd-coin-ethereum-bridged",
```

Comment:
There's no tracker for 'USDzC' in Coingecko, but I think we can use the same as for 'USDC.e'

Reply:
Is this the reason for the increased number of retried messages in the `TokenDetails` queue? I was about to investigate that today

Reply:
Should be solved after merging this.